### ISSUE TITLE
reverse wiki down

### ISSUE THREAD

**Issue by doruk100** [Android1337/New-KeyAuth-Bypass#1]
site is down

**Comment by wnelson03** [comment]
yeah, that's because he had malware on the website. you shouldn't trust him.

and the bypass doesn't work on obfuscated programs anyways. it's useless since any developer with a brain will obfuscate their program.

**Comment by Android1337** [comment]
is this a joke? it wasn't a malware it was a bypass for your trash authentication that I can easy fix and repost it
